tensorboard/plugins/scalar/vz_line_chart/vz-line-chart.ts

```diff
@@ -335,18 +405,18 @@ class LineChart {
         [[this.yAxis, this.center], [null, this.xAxis]]);
   }
 
-  private buildPlot(xAccessor, xScale, yScale): Plottable.Component {
-    this.scalarAccessor = (d: ChartHelpers.ScalarDatum) => d.scalar;
+  private buildPlot(
+      xAccessor, yAccessor, xScale, yScale, tooltipColumns): Plottable.Component {
     this.smoothedAccessor = (d: ChartHelpers.ScalarDatum) => d.smoothed;
     let linePlot = new Plottable.Plots.Line<number|Date>();
     linePlot.x(xAccessor, xScale);
-    linePlot.y(this.scalarAccessor, yScale);
+    linePlot.y(this.yValueAccessor, yScale);
     linePlot.attr(
         'stroke',
         (d: ChartHelpers.Datum, i: number, dataset: Plottable.Dataset) =>
             this.colorScale.scale(dataset.metadata().name));
     this.linePlot = linePlot;
-    let group = this.setupTooltips(linePlot);
+    let group = this.setupTooltips(linePlot, tooltipColumns);
```

Comment:
If you're changing declarations that you modify to use `const`, then you can get this one, too.